Cargo.toml

```diff
@@ -16,14 +16,13 @@ include = ["src/**/*", "Cargo.toml", "Cargo.lock", "LICENSE", "README.md", "etc/
 atty = "0.2"
 hexplay = "0.2"
 termcolor = "1"
-structopt = "0.3"
-structopt-derive = "0.4"
 rustc-demangle = "0.1"
 cpp_demangle = "0.3"
 scroll = "0.11"
 prettytable-rs = "0.8"
 env_logger = "0.9"
 anyhow = "1"
+clap = { version = "3.0.14", features = ["derive", "wrap_help"] }
```

Comment:
minor nit can you set version to `3` (instead of `3.0.14`).
Just curious, what does the `wrap_help` feature do?

Reply:
`wrap_help` feature will wrap help text when terminal size is too small to fit, since structopt also does that I've opted to enable it.